tests/api/features/boards.feature

```diff
@@ -23,7 +26,7 @@ Feature: Trello Boards API
             | desc        | a valid data table         | 
 
```

Comment:
This Scenario has no validations 🤔 (THEN step?)

Reply:
My bad! It looked like this:
![pr](https://user-images.githubusercontent.com/50670336/170577666-e6eaef3a-189a-4de2-bfb1-bda904fdfe89.jpg)
So I thought all the hidden lines were part of the same table